propolis/src/hw/bhyve/mod.rs

```diff
@@ -0,0 +1,34 @@
+mod atpic;
+mod atpit;
+mod hpet;
+mod ioapic;
+mod pmtimer;
+mod rtc;
+
+pub use atpic::*;
+pub use atpit::*;
+pub use hpet::*;
+pub use ioapic::*;
+pub use pmtimer::*;
+pub use rtc::*;
+
+use std::sync::Arc;
+
+pub type KernelDefaults = (
+    Arc<BhyveAtPic>,
+    Arc<BhyveAtPit>,
+    Arc<BhyveHpet>,
+    Arc<BhyveIoApic>,
+    Arc<BhyveRtc>,
+);
+
+/// Create cohort of devices emulated by bhyve kernel component
```

Comment:
I guess I could have advertised this fact a little more, but these are all devices which are emulated (by default) by the kernel vmm portion of bhyve.  They're here not because we're going to do emulation in userspace, but so we can gather up all their state for a migration.